### PR TITLE
Set border color to maxcontrast

### LINKED ISSUE #1254: Separator line is barely visible due to low contrast
Input fields are shown with a thin, grey line with too low contrast to the white background.

image

### PR DESCRIPTION
**Pull request by Chartman123**
This fixes #1254 by changing bottom-border color from `--color-border-dark` to `--color-text-maxcontrast`

Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Much better, although I’m not sure if it really counts as fixing the reported issue, as it’s still not soo visible.

We could merge this as it’s definitely an improvement, and then use the TextInput component as soon as it’s done? We probably need to see how it looks like e.g. when you have multiple fields below each other like for a multiselect or radio options, cause there could be too many borders floating around. cc @marcoambrosini to keep in mind maybe?

**Comment by marcoambrosini** [comment]
I'll add this style to the textField field component in a follow-up

**Comment by Chartman123** [comment]
> Much better, although I’m not sure if it really counts as fixing the reported issue, as it’s still not soo visible.

@jancborchardt We could also set the border size to 2px instead of just one. That makes the line much more visible.

File: src/components/Questions/AnswerInput.vue [hunk 1]
```diff
@@ -235,7 +235,7 @@ export default {
 	margin: 0;
 	padding: 0 0;
 	border: 0;
-	border-bottom: 1px dotted var(--color-border-dark);
```
Comment (jotoeri) [inline src/components/Questions/AnswerInput.vue:238]: Is it necessary here? As @jancborchardt already said, that looks quite hard now, if there are multiple fields below each other. Also we here have the Checkboxes as indicating element in front, while the issue mentioned the text-input fields, where this line is the input-indicator, as the checkboxes are for themselves. :thinking:

Comment (marcoambrosini) [reply src/components/Questions/AnswerInput.vue:238]: Could you post a screenshot @jotoeri please?

Comment (jotoeri) [reply src/components/Questions/AnswerInput.vue:238]: | Before | After |
|---|---|
| before | after |

Comment (Chartman123) [reply src/components/Questions/AnswerInput.vue:238]: I wouldn't mind the darker lines here. 🙈 But I could also live with the lighter ones. So I think that this should also be rated from an accessibility point of view. And perhaps to make the look more unified.

Comment (jotoeri) [reply src/components/Questions/AnswerInput.vue:238]: Anyways. Maybe i just have to geht used to it. 🙈

**Review by jotoeri — APPROVED** [review]

**Pull request merged**